landoapi/commit_message.py

```diff
@@ -185,35 +188,28 @@ def replace_first_reviewer(matchobj):
         return commit_summary.strip() + "\n" + commit_description
 
 
-def strip_commit_metadata(s):
+def strip_commit_metadata(message: str) -> str:
     """Strips metadata related to commit tracking.
 
     Will strip lines like "MozReview-Commit-ID: foo" from the commit
     message.
     """
     # TODO this parsing is overly simplified. There is room to handle
     # empty lines before the metadata.
-    lines = [l for l in s.splitlines() if not METADATA_RE.match(l)]
+    lines = [l for l in message.splitlines() if not METADATA_RE.match(l)]
 
     while lines and not lines[-1].strip():
         lines.pop(-1)
 
-    if type(s) == bytes:
-        joiner = b"\n"
-    elif type(s) == str:
-        joiner = "\n"
-    else:
-        raise TypeError("do not know type of commit message: %s" % type(s))
-
-    return joiner.join(lines)
+    return "\n".join(lines)
 
 
-def parse_commit_id(s):
+def parse_commit_id(message: str) -> Optional[str]:
```

Comment:
Is this used anywhere?

Reply:
Hm, neither of the functions you called out are used... I wonder if that's something we could configure Flake8 to look for?

In the meantime I'll remove them.